contracts/gateway.yaml

```diff
@@ -3972,11 +4086,102 @@ components:
           properties:
             purpose:
               $ref: '#/components/schemas/PatientAuthPurpose'
-            modes: 
+            modes:
               type: array
               items:
                 $ref: '#/components/schemas/AuthenticationMode'
-        error: 
+        error:
+          $ref: '#/components/schemas/Error'
+        resp:
+          $ref: '#/components/schemas/RequestReference'
+    PatientShareRequest:
+      type: object
+      required:
+        - requestId
+        - timestamp
+        - patient
+        - hipDetails
+      properties:
+        requestId:
+          type: string
+          format: uuid
+          description: a nonce, unique for each HTTP request
+          example: 5f7a535d-a3fd-416b-b069-c97d021fbacd
+        timestamp:
+          $ref: '#/components/schemas/TimestampSchema'
+        hipDetails:
```

Comment:
few things
1. In all our APIs, we usually have a single root object as payload. Lets try to keep it so.
2. Why do we need to pass the HIP ID? note, this share can be done even with HIU - right?

Reply:
Removed HipDetails part.